Shared/Supporting Files/Views/SampleDetailView.swift

```diff
@@ -101,6 +103,12 @@ struct SampleDetailView: View {
         .navigationTitle(sample.name)
         .navigationBarTitleDisplayMode(.inline)
         .toolbar {
+            ToolbarItem(placement: .topBarLeading) {
+                Button("Full Screen", systemImage: screenExpansionToggleImage) {
+                    isFullScreen.toggle()
+                    screenExpansionToggleImage = isFullScreen ? "arrow.down.forward.and.arrow.up.backward" : "arrow.up.backward.and.arrow.down.forward"
```

Comment:
A more common pattern is like [this](https://stackoverflow.com/a/72858311/14369688), to use the label parameter and an [`Image(systemName:)` ](https://developer.apple.com/documentation/swiftui/image/init(systemname:)) to toggle the button icon.

Reply:
Yeah, I thought about it that way, these strings are absurd long - arrow.down.forward.and.arrow.up.backward, arrow.up.backward.and.arrow.down.forward - so I was thinking this would make it more readable. I don't have a strong opinion either way though.